---a/net/minecraft/block/properties/PropertyHelper.java
+++b/net/minecraft/block/properties/PropertyHelper.java
@@ -2,53 +2,42 @@
 
 import com.google.common.base.Objects;
 
-public abstract class PropertyHelper implements IProperty
-{
+public abstract class PropertyHelper implements IProperty {
+
     private final Class valueClass;
     private final String name;
     private static final String __OBFID = "CL_00002018";
 
-    protected PropertyHelper(String name, Class valueClass)
-    {
+    protected PropertyHelper(String name, Class valueClass) {
         this.valueClass = valueClass;
         this.name = name;
     }
 
-    public String getName()
-    {
+    public String getName() {
         return this.name;
     }
 
-    public Class getValueClass()
-    {
+    public Class getValueClass() {
         return this.valueClass;
     }
 
-    public String toString()
-    {
+    public String toString() {
         return Objects.toStringHelper(this).add("name", this.name).add("clazz", this.valueClass).add("values", this.getAllowedValues()).toString();
     }
 
-    public boolean equals(Object p_equals_1_)
-    {
-        if (this == p_equals_1_)
-        {
+    public boolean equals(Object p_equals_1_) {
+        if (this == p_equals_1_) {
             return true;
-        }
-        else if (p_equals_1_ != null && this.getClass() == p_equals_1_.getClass())
-        {
-            PropertyHelper propertyhelper = (PropertyHelper)p_equals_1_;
+        } else if (p_equals_1_ != null && this.getClass() == p_equals_1_.getClass()) {
+            PropertyHelper propertyhelper = (PropertyHelper) p_equals_1_;
             return this.valueClass.equals(propertyhelper.valueClass) && this.name.equals(propertyhelper.name);
-        }
-        else
-        {
+        } else {
             return false;
         }
     }
 
-    public int hashCode()
-    {
+    public int hashCode() {
         return 31 * this.valueClass.hashCode() + this.name.hashCode();
     }
-}
+}
